site/domain-resource.md

```diff
@@ -0,0 +1,35 @@
+
+# Domain Resource
```

Comment:
Domain Resource -> Domain resource (Note: We've been using sentence capitalization instead of title capitalization throughout the docs.)